src/panels/lovelace/editor/hui-glance-card-editor.ts

```diff
@@ -0,0 +1,100 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
```

Comment:
Stop development of this editor in this PR please. This PR should focus on putting the foundation in place for editor elements. Having this element just edit the name is enough to validate the foundation.

Reply:
Makes sense. I have cleaned up what I have done. I took the entites out since it didnt work left the rest